edb/schema/delta.py

```diff
@@ -94,7 +96,13 @@ def delta_objects(
 
         full_matrix.append((x, y, similarity))
 
-    full_matrix.sort(key=lambda v: v[2], reverse=True)
+    full_matrix.sort(
+        key=lambda v: (
+            1.0 - v[2],
```

Comment:
Just curious, why is this better than reversed=True?

Reply:
Because sorting names in reverse is kinda weird, esp. for tests.